src/utilities/strings.ts

```diff
@@ -0,0 +1,6 @@
+export function snakeCase(string: string): string {
+    return string
+        .replace(/([a-z])([A-Z])/g, "$1-$2")
```

Comment:
it appears here like you’re putting in a ‘-’ where you maybe meant to put in a ‘_’

Reply:
that's supposed to be kebab case. Fixed 👍